libs/sdk-ui-kit/src/Header/HeaderDataMenu.tsx

```diff
@@ -56,10 +56,10 @@ export const CoreHeaderDataMenu: React.FC<IHeaderDataMenuProps> = ({
                         {tooltipText && isDisable && (
                             <Bubble
                                 alignPoints={[
-                                    { align: "bc tc" },
-                                    { align: "tc bc" },
-                                    { align: "bl tl" },
-                                    { align: "br tr" },
+                                    { align: "bc tc", offset: { x: -5, y: 0 } },
```

Comment:
Should the `offset` depend on `align`, with the current codes, it seems that `offset` is the same for all `align`

Reply:
It's different but I think we dont need to add more it here. Because it just help about when mouse hover on the border of bubbletrigger, it will be unstable to display sometimes. Thanks